packages/big-bang-core/eksctl/demo.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: eksctl.io/v1alpha5
```

Comment:
This EKSCTL seems weird here. What is it's purpose?

Reply:
Also does this even work properly with the need for addons to make storage work now?

Reply:
This is just an example of deploying an EKS cluster for testing the BigBang deployment on.  Since IronBank doesn't have amd64 images, I thought this was easier to get someone up to a consistent baseline for testing.   See the updates to the Readme.md for where it's being used